src/key_events.h

```diff
@@ -43,6 +43,30 @@ extern const Key keymaps[][ROWS][COLS];
  */
 void handleKeyswitchEvent(Key mappedKey, byte row, byte col, uint8_t keyState);
 
-// Internal use
+/** Flushes any pending regular key switch events and sends them out */
+void sendKeyboardReport();
+
+// A facade on top of our HID implementation
 void pressKey(Key mappedKey);
 void releaseKey(Key mappedKey);
+void releaseAllKeys();
+void pressKeyRaw(Key mappedKey);
+void releaseKeyRaw(Key mappedKey);
+
+void pressConsumer(Key mappedKey);
+void releaseConsumer(Key mappedKey);
+
+void pressSystem(Key mappedKey);
+void releaseSystem(Key mappedKey);
+
```

Comment:
While this sounds okay-ish, `pressSystemKey` conveys the intent of the functions better for me.

Or, if `pressKeyRaw` remains that, then this should likely be `pressKeySystem`, and the two above `pressKeyConsumer` - and that would feel wrong. So I'm starting to think that `pressKeyRaw` should be `pressRawKey`, and these should be `pressSystemKey` and `pressConsumerKey` above.